Rdkafka.xs

```diff
@@ -393,11 +393,17 @@ krdm_offset(msg)
         RETVAL
 
 long
-krdm_timestamp(msg)
+krdm_timestamp(msg,...)
         rd_kafka_message_t* msg
     CODE:
 	rd_kafka_timestamp_type_t tstype;
         RETVAL = rd_kafka_message_timestamp(msg, &tstype);
+	if (items > 1) {
+	    if (!SvROK(ST(1)) || strncmp(sv_reftype(SvRV(ST(1)), 0), "SCALAR", 7)) {
```

Comment:
Can/should we use `SvTYPE(SvRV(ST(1))) < SVt_PVAV` to avoid the string compare?

Reply:
I would like to avoid the string compare --- I just copied similarily looking code (like https://github.com/plicease/Kafka-Librd/blob/master/Rdkafka.xs#L118 , and there are more of this).